apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -509,22 +509,39 @@ emqx_schema {
     }
   }
 
-  persistent_session_store_storage_type {
-    desc {
-        en: "Store information about persistent sessions on disc or in ram.\n"
-            "If ram is chosen, all information about persistent sessions remains\n"
-            "as long as at least one node in a cluster is alive to keep the information.\n"
-            "If disc is chosen, the information is persisted on disc and will survive\n"
-            "cluster restart, at the price of more disc usage and less throughput.\n"
-        zh: "将有关持久会话的信息存储在磁盘或内存中。\n"
-            "如果选择了ram，有关持久会话的所有信息将保留\n"
-            "只要群集中至少有一个节点处于活动状态，就可以保留信息。\n"
-            "如果选择了光盘，则信息将保留在光盘上，并且将继续存在\n"
-            "群集重新启动，代价是磁盘使用量增加，吞吐量降低。\n"
-    }
-    label: {
-        en: "Storage type"
-        zh: "存储类型"
+  persistent_session_store_backend {
+    desc {
+        en: "Database backend used to store information about the persistent sessions.\n"
+            "- `builtin`: Use an embedded database (mria)"
+        zh: "用于存储持久性会话信息的数据库后端\n"
```

Comment:
Warning: machine translation

Reply:
I modified parts of it to be it looks more localized (maybe). it looks like each one configuration needs a lot of explanation if it needs to be clearly understood. 😭

Reply:
Thanks for the comment, I will make descriptions more in-depth.

Reply:
Does it make more sense now https://github.com/emqx/emqx/pull/7897/commits/0f8829dea1e929e3d1d6439a7d1d80a8ffc9639b ?